TrustModel.md

```diff
@@ -0,0 +1,426 @@
+the runtime environment in which the workload is truely confidential. Threats can exist during 
+these lifecycle phases and trust must be considered in the context of the lifecycle of the 
+environment supporting confidential containers.
+
+
+![Lifecycle Phases](./images/LifecyclePhases.svg)
+
+Taking the four phases above we can map this as :
+
+#### Develop
+ - Preparation and build of container images
+ - Preparation and build of the trusted boot image for the TEE. 
+ - Signatures and encryption keys for containers are prepared and available
+ 
+````` Completing building the boot image triggers the next phase.`````
+
+#### Distribute
+The Distribute stage includes all preparations before the guest executes its first instructions, 
+and it will run some general and TEE-specific operations.
+- configuring the cluster with TEE
+- making the boot image available
+- taking initial measurements
+- provision secrets.
+- allocation of guest resource
+- loading guest fw and configuration occurs 
+- TEE-specific steps relating to preparing attestation/secrets etc
+
+The involved components include the ones that directly associated with the
+protections that TEEs provide (e.g, tdx seam-loader, tdx module, psp/sev firmware and so on),
+Pre-OS (e.g, guest firmware, guest kernel, initrd and so on) and specific components (e.g,
+kata-agent, attestation-agent and so on). Also includes preparing secrets for delivery to the TEE.
+
+```The first instruction being executed within the trusted boot image happens in the next phase```
+
+#### Deploy
+  - pod sandbox preparation
+  - "pre flight checks" include attestation measurment checks
+  - secret provisioning linked to boot attestation
+  
+  ```Receiving the request to pull and deploy a container happens in the runtime phase```
+
+
+#### Runtime
+- post boot attestation and secret provisioning
+- procedure of image pulling, unpacking and 
+  launching.
+- the workload in the container starts to  run in TEEs.
+
+
+These execution phases are sufficient to describe the execution of any forms of confidential
+container. 
+
+---
+---
+
+## Goals and not-goals
+
+### Goals
+
+The commonality of goals between confidential container and confidential computing are to reduce
+the ability for the platform owner, operator and attacker to access data and code inside TEEs
+sufficiently such that this path is not an economically or logically viable attack during
+execution[^1]  5.1 Goal.
+
+Around the three execution stages of confidential container, the scope of the threat model of
+confidential container includes:
+
+<* ADD diagram*>
+
+- Pre-container stage
+  - Measure the TEE where confidential containers run, and store the measurements to a secure location.
+
+- Container launch stage
+  - Measure and attest the TEE where confidential containers run to prove the deployment and initialization of pre-container are valid and expected;
```

Comment:
Measurement is listed in both of these stages (Pre-container and Container launch) but I think measurement, at least from the HW, only happens once.